src/main/service.js

```diff
@@ -362,10 +362,13 @@ function getMetaQuestionForAppendix(req, res, next){
  */
 function addMetaQuestion(req, res, next){
     try{
-        let metaQuestion = application.addMetaQuestion(req.body)
-        req.log.info("request to create metaQuestion");
-        res.send(200, {code:200, metaQuestion})
-        next()
+        application.addMetaQuestion(req.body).then(
+            metaQuestion => {
+                req.log.info("request to create metaQuestion");
+                res.send(200, {code:200, metaQuestion})
+                next()
+            }
+        )
```

Comment:
you don't encapsulate an `async` call in `try/catch`
instead you pass it 2 callbacks: 
1. first for success, the argument it takes is the returned value (undefined for void funcs).
2. second for if an error occurs, the argument is the thrown error. 

take a look at other `then`s I implemented in this file